src/Symfony/Component/DependencyInjection/Compiler/PriorityTaggedServiceTrait.php

```diff
@@ -76,6 +76,8 @@ private function findAndSortTaggedServices($tagName, ContainerBuilder $container
 
                 if (null !== $indexAttribute && isset($attribute[$indexAttribute])) {
                     $index = $attribute[$indexAttribute];
+                } elseif (isset($attribute['name'])) {
```

Comment:
We cannot have a `name` attribute in DI tags, because YAML and XML are reserving `name` for the name of the tag itself. so this is a bad idea to me

Reply:
You might have missed https://github.com/symfony/symfony/pull/36586 ;)

Reply:
indeed :smile: